### PR TITLE
Add gnome keyring

### PR DESCRIPTION
**Pull request by saw-jan**
Added `gnome-keyring` keyring provider.

Test logs with:
 - image WITHOUT gnome-keyring
 ```
[ warning sync.credentials.http.legacy ]:	Migrating old keychain entries failed "The name org.freedesktop.secrets was not provided by any .service files"
```
- image WITH gnome-keyring
```
[ warning sync.credentials.http.legacy ]:	Migrating old keychain entries failed "Entry not found"
```

### CODE REVIEW

**Comment by saw-jan** [comment]
Test logs with:
 - image WITHOUT gnome-keyring
 ```
[ warning sync.credentials.http.legacy ]:	Migrating old keychain entries failed "The name org.freedesktop.secrets was not provided by any .service files"
```
- image WITH gnome-keyring
```
[ warning sync.credentials.http.legacy ]:	Migrating old keychain entries failed "Entry not found"
```

File: fedora/src/startup/vnc_startup.sh [hunk 1]
```diff
@@ -93,41 +93,41 @@ chmod 600 "${PASSWD_PATH}"
...
-    echo "Saving default startup script as ${XSTARTUP_FILE}.old"
     cp "${XSTARTUP_FILE}" "${XSTARTUP_FILE}.old"
     echo "Replacing default startup script ${XSTARTUP_FILE}"
     cat <<'EOF' > "${XSTARTUP_FILE}"
 #!/bin/sh
 unset SESSION_MANAGER
 unset DBUS_SESSION_BUS_ADDRESS
+# unlock keyring
+echo -n ${VNC_PW} | gnome-keyring-daemon --unlock
 startxfce4 &
```
Comment (saw-jan) [inline fedora/src/startup/vnc_startup.sh:107]: unlock keyring before `startxfce4`.
Also, is it needed to start secrets service with `gnome-keyring-daemon --start --components=secrets` or desktop-client will handle it?

CC @fmoc

**Review by fmoc — COMMENTED** [review]
Looks okay to me. Have not tested it myself.

**Comment by saw-jan** [comment]
we might not need this

**Pull request closed without merging**